sacc/tracers.py

```diff
@@ -216,17 +521,32 @@ class NZTracer(BaseTracer, tracer_type='NZ'):
 
     Attributes
     ----------
```

Comment:
I guess the code has been doing this wrong consistently. These are parameters.

Reply:
Yep. I've tried to fix this.

Reply:
Matt is still wrong here.